src/index.html

```diff
@@ -11,8 +11,57 @@
       rel="stylesheet"
       href="./styles/index.scss"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wdth,wght@0,75..100,100..900;1,75..100,100..900&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <div class="card__image">
+        <img
+          src="./images/imac.jpeg"
+          alt="mac"
+        />
+      </div>
+      <div class="card__text">
+        APPLE A1419 iMac 27" Retina
+        <br />
+        5K Monoblock (MNED2UA/A)
+      </div>
+      <div class="card__code">Product code: 195434</div>
```

Comment:
Why did you use div for text everywhere? You need to use the semantic tag, for example `h3, p`